runpod/serverless/modules/rp_job.py

```diff
@@ -134,4 +157,4 @@ def run_job_generator(
     finally:
         log.info(f'{job["id"]} | Finished ')
 
-        return None  # pylint: disable=lost-exception
+        # return None  # pylint: disable=lost-exception
```

Comment:
I can't remember why I had this structured like this, should be able to remove this line entirely if not required.